packages/app/features/player/components/likesList.tsx

```diff
@@ -1,15 +1,23 @@
-import { User } from "app/api/graphql";
+import { Entry, PublicUser, useEntryLikesQuery } from "app/api/graphql";
 import { Text, View } from "app/design-system";
-import { FavoriteButton } from "app/ui/buttons/favoriteButton";
+import { LikeButton } from "app/ui/buttons/likeButton";
 import { UserAvatar } from "app/ui/userAvatar";
 import { FlatList } from "react-native";
+import { isSome } from "app/utils";
 
 type Props = {
-  likers: User[];
+  entry: Entry;
+  classname?: string;
 };
 
-export function LikesList({ likers }: Props) {
-  const renderItem = ({ item }: { item: User }) => {
+export function LikesList({ entry, classname = "" }: Props) {
+  const { data } = useEntryLikesQuery({
+    variables: {
+      id: entry.id!,
+    },
+  });
+
```

Comment:
we need some state when the list is loading and empty.

Reply:
Doing it in #33

Reply:
can you do it here? #33 is about the collection feature, so we shouldn't include in it things connected with likes.

Reply:
I extracted list component and using it both for likes and collections

Reply:
Wait, I probably confused lists

Reply:
Yes you are right, I was thinking about different list